aries_cloudagent/commands/upgrade.py

```diff
@@ -136,68 +168,64 @@ async def upgrade(settings: dict):
                         f"{upgrade_from_version} as --from-version from "
                         "the config."
                     )
-        if upgrade_from_version == upgrade_to_version:
+        upgrade_version_in_config = get_upgrade_version_list(
+            sorted_versions_found_in_config, upgrade_from_version
+        )
+        force_upgrade_flag = root_profile.settings.get("upgrade.force_upgrade") or False
+        if upgrade_from_version == upgrade_to_version and not force_upgrade_flag:
             print(
                 f"Version {upgrade_from_version} to upgrade from and "
                 f"current version to upgrade to {upgrade_to_version} "
-                "are same."
+                "are same. If you still wish to run upgrade then plese "
```

Comment:
Typo in “please”. 

It makes no sense to me to force and upgrade when the “from” and “to” are the same, so I don’t think the logic here is right.  I think that it should just print a message a proceed.  An upgrade is only possible when the `from` is before the `to` and there are upgrades in between the two versions.